packages/cli/test/unit/util/dev/server.test.ts

```diff
@@ -1,438 +0,0 @@
-import ms from 'ms';
-import url from 'url';
-import path from 'path';
-import execa from 'execa';
-import fs from 'fs-extra';
-import fetch, { Response } from 'node-fetch';
-import listen from 'async-listen';
-import { createServer } from 'http';
-import { client } from '../../../mocks/client';
-import DevServer from '../../../../src/util/dev/server';
-import { DevServerOptions } from '../../../../src/util/dev/types';
-
-const IS_WINDOWS = process.platform === 'win32';
-
-async function runNpmInstall(fixturePath: string) {
-  if (await fs.pathExists(path.join(fixturePath, 'package.json'))) {
-    return execa('yarn', ['install', '--network-timeout', '1000000'], {
-      cwd: fixturePath,
-      shell: true,
-    });
-  }
-}
-
-interface TestFixtureOptions extends Omit<DevServerOptions, 'output'> {
```

Comment:
Why was this deleted?

Reply:
Carry-over from #7845. Not included in the diff here anymore since that PR got merged.